metadata-ingestion/src/datahub/ingestion/source/looker/looker_lib_wrapper.py

```diff
@@ -197,6 +202,27 @@ def all_dashboards(self, fields: Union[str, List[str]]) -> Sequence[DashboardBas
             transport_options=self.transport_options,
         )
 
+    def all_looks(self, fields: Union[str, List[str]]) -> List[Look]:
+        self.client_stats.all_looks_calls += 1
+        looks: List[Look] = list(
+            self.client.all_looks(
+                fields=self.__fields_mapper(fields),
+                transport_options=self.transport_options,
+            )
+        )
+        # Add soft deleted looks
+        looks.extend(self.search_looks(fields=fields, deleted=True))
```

Comment:
why are we including soft-deleted looks?

Reply:
in that case, we should pass this in as a method parameter and only set deleted=True if included_deleted is set